.builds/freebsd.yml

```diff
@@ -16,3 +17,7 @@ tasks:
     cd hidapi
     make
     make DESTDIR=$PWD/root install
+    make clean
+- build-manual: |
+    cd hidapi/libusb
+    gmake -f Makefile-manual
```

Comment:
Unfortunately the freebsd manual makefiles aren't posix compliant, so gmake is needed there. I researched a bit but I didn't find a good reference how e.g. `ifeq` statements should look like in POSIX make, so let's just use gmake